src/utils.ts

```diff
@@ -76,6 +77,29 @@ function handleLegacyTokens(token: string): Token {
 	return { token: [{ proofs: obj.proofs, mint: obj?.mints[0]?.url ?? '' }] };
 }
 
+export function isObj(v: unknown): v is object {
+	return typeof v === 'object';
+}
+
+export function checkResponse(data: { error?: string; detail?: string }) {
```

Comment:
I haven't tested the behaviour, and maybe i'm wrong, but wouldn't this only throw the `error` , if both `detail` and `error` are present? would it make sense to accumulate them?

Reply:
This will throw if detail or error is present. I don't know of a case where both will present.

Reply:
I think it shouldn't be an issue for now. if it is, we will find out later 👍